ModulemdTranslationHelpers/cli.py

```diff
@@ -71,26 +70,32 @@ def cli(ctx, debug, branch, koji_url):
 
 
 @cli.command()
+
 @click.option('-p', '--pot-file',
               default='fedora-modularity-translations.pot',
               type=click.File(mode='wb', atomic=True, lazy=True),
               show_default=True,
               metavar="<PATH>",
               help="Path to the portable object template (POT) file to hold "
                    "the translatable strings.")
+
+@click.option('--project-name', prompt='Project name',
```

Comment:
Please don't prompt for this. If unspecified, it's better for it to just take the default.